website/docs/docs/dbt-cloud/cloud-ide/the-ide-git-button.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "The IDE Git Button"
+---
+
+The git button in the IDE allows you to apply the concept of [version control](dbt-cloud/cloud-ide/the-dbt-ide#version-control) to your project. This page provides descriptions of each git button command and what they do.
+
+**Initialize your project**: This is done when first setting up your project. Initializing a project will create all required directories and files within an empty repository. Note: This option will not display if your repo isn't completely empty (i.e. includes a README file).
+
+<p align="center">
+<Lightbox src="/img/initialize.png" />
```

Comment:
re: the images are broken

You'll have to add the images to the repo as well — I think I go over that in the Loom vid in the README :)

For example, if I have an image reference like this:
```
<Lightbox src="/img/create-bigquery-project.png" title="Create a new GCP project" />
```
It picks up the image here: https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static/img

Reply:
ah tysm!